docs/csharp/whats-new/csharp-7-3.md

```diff
@@ -0,0 +1,189 @@
+---
+title: What's new in C# 7.3
+description: An overview of new features in C# 7.3
+ms.date: 05/16/2018
+---
+# What's new in C# 7.3
+
+There are two main themes to the C# 7.3 release. One theme provides features that enable safe code to be as performant as unsafe code. The second theme provides incremental improvements to existing features. In addition, new compilers were added in this release.
+
+The following new features support the theme of better performance for safe code:
+
+- You can access fixed fields without pinning.
+- You can reassign `ref` local variables.
+- You can use initializers on `stackalloc` arrays.
+- You can use `fixed` statements with any type that supports a pattern.
+- You can use additional generic constraints.
+
+The following enhancements were made to existing features:
+
+- You can test `==` and `!=` with tuple types.
+- You can use expression variables in more locations.
+- You may attach attributes to the backing field of auto implemented properties.
+- Method resolution when arguments differ by `in` has been improved.
+- Overload resolution new has fewer ambiguous cases.
+
+The new compiler options are:
+
+- `deterministic` to ensure that subsequent compilations of the same source generate the same binary output.
```

Comment:
I'm surprised. "deterministic" is not a new option. It was there for a while.

Reply:
Well, we got behind. I can move it to the correct version. Do you know when it was first added?

Reply:
`-deterministic` was added in C# 6.0.